graph-codegen/src/settings/resource_settings.rs

```diff
@@ -1967,6 +1974,21 @@ pub fn get_write_configuration(resource_identity: ResourceIdentity) -> WriteConf
 			.filter_path(vec!["workbook", "getActivitiesByInterval()"])
 			.build()
 			.unwrap(),
+		ResourceIdentity::Workbook => WriteConfiguration::second_level_builder(ResourceIdentity::Drives, resource_identity)
+			.trim_path_start("/drives/{drive-id}/items/{driveItem-id}")
+			.filter_path(vec!["worksheets", "tables", "functions", "names", "comments"])
+			.build()
+			.unwrap(),
+		ResourceIdentity::Worksheets => WriteConfiguration::second_level_builder(ResourceIdentity::Drives, resource_identity)
+			.trim_path_start("/drives/{drive-id}/items/{driveItem-id}/workbook")
+			.filter_path(vec!["usedRange","tables", "charts", "names", "pivotTables"])
+			.build()
+			.unwrap(),
+		ResourceIdentity::WorksheetsUsedRange => WriteConfiguration::second_level_builder(ResourceIdentity::Drives, resource_identity)
```

Comment:
Needed to create separate resource for used range since there were method names conflict just like described in wiki with $count